core/src/pending_collision.rs

```diff
@@ -0,0 +1,240 @@
+use bevy::{prelude::*, render::mesh::VertexAttributeValues};
+
+use crate::{CollisionShape, RigidBody};
+
+/// Component which indicates that this enityty contains scene(s) which waiting for collision generation.
+///
+/// Once the scene is instantiated (Bevy loads the scenes asynchronously), then all elements of the scene will be added [`RigidBody`] and [`CollisionShape::ConvexHull`] (based on the geometry) components.
+///
+/// # Example
+///
+/// ```
+/// # use bevy::prelude::*;
+/// # use heron_core::*;
+/// fn spawn(mut commands: Commands, asset_server: ResMut<AssetServer>) {
+///     commands
+///         .spawn()
+///         .insert(Transform::default())
+///         .insert(GlobalTransform::default())
+///         .insert(PendingConvexCollision {
+///             body_type: RigidBody::Static,
+///             border_radius: None,
+///         })
+///         .with_children(|parent| {
+///             parent.spawn_scene(asset_server.load("cubes.glb#Scene0"));
```

Comment:
why does the mesh handle need to be in a child?

Reply:
Because you can't insert a component to a scene. I.e. `commands.spawn_scene("model").insert(PendingConvexCollision)` won't work.

Reply:
ah. ok. indeed.